src/main/java/com/puppycrawl/tools/checkstyle/checks/FinalParametersCheck.java

```diff
@@ -68,13 +68,71 @@
  * &lt;module name=&quot;FinalParameters&quot;/&gt;
  * </pre>
  * <p>
+ * Example :
+ * </p>
+ * <pre>
+ * public class Point {
+ *       private int m,n;
+ *       private String s;
+ *       public Point(){ //ok
+ *       s = "ABC";
+ *       }
+ *       public Point(final int m) { // ok
+ *           this();
+ *           this.m = m;
+ *       }
+ *       public Point(final int m,int n){ //Violation - n should be final
```

Comment:
make space after `//` and lovercacse first word. Please apply this for all lines.

`// violation - n should be final`